stable/datacube-ows/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+description: Datacube Web Map Service
+name: datacube-ows
+version: 0.18.3
+keywords:
+- datacube
+- http
+- web
+- application
+- python
+home: https://www.opendatacube.org/documentation
+# icon:
+sources:
+- https://github.com/opendatacube/datacube-core
+- https://github.com/opendatacube/datacube-ows
+# maintainers:
```

Comment:
@pindge please take ownership as maintainer of this Chart 👍

Reply:
lol? really?